src/main/web_src/src/components/fagsystem/tpsf/form/familierelasjoner/partials/Barn.js

```diff
@@ -38,6 +38,15 @@ export const initialValuesDoedfoedt = {
 }
 
 export const Barn = ({ formikBag, personFoerLeggTil }) => {
+	const filterBorHos = options => {
+		const partnere = formikBag?.values?.tpsf?.relasjoner?.partnere
+		return partnere
+			? options.filter(opt =>
+					partnere[0].harFellesAdresse ? opt.value !== 'BEGGE' : opt.value !== 'OSS'
```

Comment:
Holder dessverre ikke å sjekke den første partneren, for en person kan ha mange partnere 😢  Tror det skal funke bedre om du sjekker hvilken partner som er forelder først (men mulig det kan bli litt tricky om formet ikke blir fylt ut i "riktig" rekkefølge) 🤔  Si ifra om det er noe jeg kan prøve å hjelpe til med 😊

Reply:
Hmm, skjønner 😅  Skal prøve litt å se, så sier jeg ifra hvis det blir litt for tricky 😄